### PR TITLE
Added installation, contributing, license and credits

### PR DESCRIPTION
**Pull request by Kierst01**
Hi, I added a few more sections to the readme. Please have a look and let me know if changes should be made. Thanks!

### CODE REVIEW

**Review by rorywhite200 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
-This package provides password management tools in Python. The package consists of four functions. The first function is responsible for generating passwords, while the second evaluates their strength, categorizing them as weak, medium, or strong. The third function handles password encryption, ensuring data security, and the fourth function allows for decryption when necessary. 
 
-## List of functions
+This package provides password management tools in Python. The package consists of four functions. The first function is responsible for generating passwords, while the second evaluates their strength, categorizing them as weak, good, or strong. The third function handles password encryption, ensuring data security, and the fourth function allows for decryption when necessary. 
+
+Functions included in this package:
 - password_creator
 - password_strength
 - password_encryption
 - password_decryption
```
Comment (rorywhite200) [inline README.md]:
```suggestion
- generate_password
- password_strength
- password_encryption
- decrypt_password
```

**Review by rorywhite200 — CHANGES_REQUESTED** [review]

**Comment by rorywhite200** [comment]
Hi @Kierst01 I approved your changes and altered the names of the functions in the documentation to match those in our src folder. Someone else would need to approve again

**Review by zywkloo — APPROVED** [review]
Look good to me.
I think I will change `password_encryption` to `encrypt_password` in a new PR, to align with `decrypt_password`

**Review by zywkloo — APPROVED** [review]

**Comment by rorywhite200** [comment]
@zywkloo Merging can be performed automatically once the requested changes are addressed.

**Review by mishelly-h — APPROVED** [review]
lgtm

**Review by mishelly-h — APPROVED** [review]
looks good

**Comment by rorywhite200** [comment]
Merging is still blocked - do you see a way to accept the requested changes?

**Review by rorywhite200 — APPROVED** [review]

**Comment by rorywhite200** [comment]
> Merging is still blocked - do you see a way to accept the requested changes?

Got it now

**Pull request merged**